### ISSUE TITLE
How to restrict the number of sliding pages (with a single swipe)?

### ISSUE THREAD

**Issue by bb-in-hoodie** [naver/egjs-flicking#603]
## Description

I cannot find an option to restrict the number of moving slides. What I want is to make it slides only a single page by a single swipe. (not making it slides multiple pages)

**Comment by WoodNeck** [comment]
Hello @bb-in-hoodie!
It looks like our document is missing, thanks for the information! I'll fix that asap.

You can achieve that by setting `moveType: "strict"`.
And, you can control the number of slides to pass in one interaction like this:
```js
// simple
// It's same as ["strict", { count: 1 }]
new Flicking(el, {
  moveType: "strict"
})

// detailed
new Flicking(el, {
  moveType: ["strict", { count: 3 }]
});

// And you can also use our predefined constants
import { MOVE_TYPE } from "@egjs/flicking"

new Flicking(el, {
  moveType: MOVE_TYPE.STRICT // "strict"
})

```

**Comment by bb-in-hoodie** [comment]
How'd you even miss that important option from the docs in the first place! 😅
It works so well, thanks for your help 👍

**Comment by WoodNeck** [comment]
Document will be fixed in #628
